packages/protocol/contracts/identity/Attestations.sol

```diff
@@ -779,7 +797,9 @@ contract Attestations is
 
     IRandom random = IRandom(registry.getAddressForOrDie(RANDOM_REGISTRY_ID));
 
-    bytes32 seed = random.random();
+    bytes32 seed = random.getBlockRandomness(
+      unselectedRequest.blockNumber + selectIssuersWaitBlocks
```

Comment:
Assuming this is what enforces the wait for us?

Reply:
Yup! Implicitly it enforces it assuming that future randomness is not accessible